app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
+    def __init__(self, nickname: str, favourite_dish: str):
+        super(Dwarf, self).__init__(nickname)

```

Comment:
Here and in other files, not necessary to use "Dwarf, self" inside "super". But it is also up to you.